src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -7753,3 +7753,15 @@ def _output(self, *args, **kwargs):
             item['localName'] = item['name']['localizedValue']
         return result, next_link
 # endregion
+
+
+def remove_nw_connection_monitor_output(cmd, connection_monitor_name, location):
+
+    update_args = {
+        'connection_monitor_name': connection_monitor_name,
+        'location': location
+    }
+    from azure.cli.core.commands import LongRunningOperation
+    from .operations.watcher import WatcherConnectionMonitorOutputRemove
+    poller = WatcherConnectionMonitorOutputRemove(cli_ctx=cmd.cli_ctx)(command_args=update_args)
+    return LongRunningOperation(cmd.cli_ctx)(poller)
```

Comment:
You can directly return the poller

```suggestion
    return WatcherConnectionMonitorOutputRemove(cli_ctx=cmd.cli_ctx)(command_args=update_args)
```